.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,80 @@
+---
+name: Problem report
+about: Create an extensive report to help us document a problem
+
+---
+<!--- Please fill out this template to the best of your ability. You can always edit this issue once you have created it. -->
+<!--- Read the following link before you create a new problem report: https://kodi.wiki/view/HOW-TO:Submit_a_bug_report  -->
+## Bug report
+### Describe the bug
+Here is a clear and concise description of what the problem is:
+<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+<!--- A bug report that is not clear will be closed -->
+<!--- Put your text below this line -->
+
+
+
+## Expected Behavior
+Here is a clear and concise description of what was expected to happen:
+<!--- Tell us what should happen -->
+<!--- Put your text below this line -->
+
+
+
+## Actual Behavior
+<!--- Tell us what happens instead -->
+<!--- Put your text below this line -->
+
+
+
+## Possible Fix
+<!--- Not obligatory, but suggest a fix or reason for the bug -->
+<!--- Put your text below this line -->
+
+
+
+### To Reproduce
+Steps to reproduce the behavior:
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+<!--- Put your text below this line -->
+1. 
+2.
+3.
+
+
+### Debuglog
+<!--- Put your text below this line -->
+<!--- A debuglog is always mandatory when creating an issue. Provide one! -->
+The debuglog can be found here:
```

Comment:
@poisdeux @SyncedSynapse Not sure if this is wanted or not. You have any ideas/suggestions?

Reply:
I think this should be optional. Most users reporting issues will not be able to provide a debug log. 
I suggest to make only *Expected Behavior*, *Actual Behavior*, *Steps to reproduce*, and *Your Environment* mandatory. All other sections can be optional.

Reply:
We just added them two weeks ago and have not really gotten many tickets yet as we have not advertised them as such. This is mostly a copy of another project that had it like this. I'm fine reducing the template a bit more as long as we get the info we need. So open for suggestions and we can fix/use the same template in all our projects

Reply:
I don't want to necessarily reduce the template - the structure is fine IMO - i just don't want to introduce too much friction in the bug reporting process by including too many mandatory sections. But i'm fine with having them on the template. 
Anyway, on a second look, the sections i indicated aren't described as mandatory, so i'm fine with it. Including the log shouldn't also be mandatory on Kore (this is probably a difference with Kodi), as we don't have much use for it in the majority of bugs.
So to summarize, i'm fine with this, as long as the log isn't.
My only suggestion would be regarding the order of the sections. Maybe ordering them by their "mandatory" status would be helpful, so i would put the _Your system_ upwards (maybe even the 2nd section) and switch _Steps to reproduce_ with _Possible fix_. If this has been discussed elsewhere and agreed upon, disregard this, it's not very relevant.
- 
-